backend/graph/service/student.go

```diff
@@ -211,3 +212,16 @@ func (ss *studentService) UpdateStudent(newStudent model.NewStudentFields) (*mod
 
 	return model.ConvertStudent(&student), nil
 }
+
+func (ss *studentService) DeleteStudent(id string) (bool, error) {
+	var student db.Student
+	err := ss.db.First(&student, id).Error
+	if err != nil {
```

Comment:
下に合わせてこれもワンライナーで書いていいんじゃね
```suggestion
	if err := ss.db.First(&student, id).Error; err != nil {
```